src/bin/server/temp.rs

```diff
@@ -0,0 +1,60 @@
+use std::env;
+use std::fs;
+use std::io;
+use std::path::PathBuf;
+
+use rand;
+use rand::distributions;
+use rand::Rng;
+
+/// Creates files with randomly generated file names in a temporary directory.
+pub struct TemporaryFiles {
+    /// Path where files are stored.
+    root: PathBuf,
+}
+
+impl TemporaryFiles {
+    pub fn new(suffix: &str) -> io::Result<TemporaryFiles> {
+        let mut temp_dir = env::temp_dir();
+        temp_dir.push(suffix);
+
+        fs::create_dir_all(temp_dir.as_path())?;
+
+        Ok(TemporaryFiles { root: temp_dir })
+    }
+
+    /// Gets path inside the root directory of a randomly named file.
+    pub fn get_file(&self) -> PathBuf {
```

Comment:
This does not "get" a file, it creates a new file path (although it does not make the actual file).

Reply:
`get_file_path`?